src/components/ToolbarButton.tsx

```diff
@@ -49,17 +49,18 @@ const ToolbarButtonComponent: FC<DraggableToolbarButtonProps> = ({
   if (!shortcut) {
     throw new Error('Missing shortcut: ' + shortcutId)
   }
-  const { svg, exec, isActive, canExecute } = shortcut
+  const { svg, exec, canExecute } = shortcut
 
   if (!svg) {
     throw new Error('The svg property is required to render a shortcut in the Toolbar. ' + shortcutId)
   }
 
   const commandState = commandStateStore.useSelector(state => state[shortcutId])
+
   const isDraggingAny = useSelector(state => !!state.dragShortcut)
   const dragShortcutZone = useSelector(state => state.dragShortcutZone)
-  const isButtonActive =
-    useSelector(state => (customize ? selected : !isActive || isActive(() => state))) || (!customize && commandState)
+  const isButtonActive = useSelector(state => (customize ? selected : commandState))
```

Comment:
You have deleted the `customize` logic. When in toolbar customization mode, the active state is set to `selected`.

You can test this out by going to Settings → Customize Toolbar and selecting a toolbar button.

Reply:
I fixed this issue. Works for both the customize and the regular tool bar. There logic is much simpler :)